packages/cards-editor-middleware/src/index.ts

```diff
@@ -0,0 +1,130 @@
+import type { RequestHandler, Request, Response } from 'express';
+import type { MiddlewareParameters } from '@ui5/server';
+import { json, Router as createRouter } from 'express';
+import path, { join } from 'path';
+import { existsSync, writeFileSync, mkdirSync, readFileSync } from 'fs';
```

Comment:
Can we please switch to asynchronous read/write?
Like
```typescript 
import { promises } from 'fs';
..
await promises.writeFile(path, content);
```

Reply:
done